src/main/webapp/resources/js/components/ncbi/details/NcbiExportDetailsView.tsx

```diff
@@ -16,24 +19,25 @@ import { DataFunctionArgs } from "@remix-run/router/utils";
 import { SPACE_LG } from "../../../styles/spacing";
 import { BasicListItem } from "../../lists/BasicList";
 
+type LoaderType = [BasicListItem[], BioSampleDetails[]];
+
 /**
  * React router data loader (https://beta.reactrouter.com/en/dev/route/loader)
  * Fetches the submission details and formats them to be used by the component.
  * @param params
  */
 export async function loader({
   params,
-}: DataFunctionArgs): Promise<[BasicListItem[], BioSampleFileDetails[]]> {
+}: DataFunctionArgs): Promise<LoaderType> {
   const { id, projectId } = params;
   if (projectId && id) {
     return getNcbiSubmission(parseInt(projectId), parseInt(id)).then(
-      (
-        submission: NcbiSubmission
-      ): [BasicListItem[], BioSampleFileDetails[]] => {
-        const { bioSampleFiles, ...info } = submission;
+      (submission: NcbiSubmission): LoaderType => {
+        const { bioSamples, ...info } = submission;
         const details = formatNcbiSubmissionDetails(info);
-        const bioSamples = formatNcbiBioSampleFiles(bioSampleFiles);
-        return [details, bioSamples];
+        const bioSampleDetails = formatNcbiBioSampleFiles(bioSamples);
+        console.log({ bioSampleDetails, details });
```

Comment:
Remove logging statement

Reply:
Updated in 0665c8164533682af6f3f9a1177a9e31dc3b6123